doc/sidewalk_known_issues.rst

```diff
@@ -8,37 +8,44 @@ See the list of known issues that are valid for the current state of development
 v1.14.5-dev1
 ------------
 
-* KRKNWK-17374: Zephyr fatal error (assertion in semaphore module) after link down on FSK.
-    The issue occurs in Sidewalk 1.14 libraries, will be fixed in 1.16.
-    It reproduces rarely - once per few days.
-    In release mode device resets automatically, in debug mode it need to be reset manually.
+* KRKNWK-17374: Zephyr fatal error after enabling downlink on FSK
+    After enabling downlink on FSK, Zephyr fatal error occurrs due to assertion in semaphore module.
+    The error reproduces rarely - once per few days.
+    In the release mode the device resets automatically, however, in the debug mode it need to be reset manually.
+    Currently, this issue occurs for Sidewalk 1.14 libraries and it will be fixed in the version 1.16.
 
-* KRKNWK-17320: Bluetooth LE connection between Gateway and End Node drops within 30 seconds of being idle.
-    The issue occurs only on the nRF5340 development kit (experimental support) and happens due to failing certification test.
+* KRKNWK-17320: Bluetooth LE connection between Gateway and End Node drops within 30 seconds of being idle
+    The issue happens due to failing certification test.
```

Comment:
For these 2 items (line 17 and 22) is there any additional description that can be provided?

Reply:
The issue KRKNWK-17320 is solved, and can be removed